apps/designer/app/designer/features/style-panel/render-property.tsx

```diff
@@ -177,27 +184,51 @@ const ComboboxControl = ({
       <PropertyName property={styleConfig.property} label={styleConfig.label} />
       <Flex align="center" css={{ gridColumn: "2/4" }} gap="1">
         <Combobox
-          id={styleConfig.property}
+          name="prop"
           items={styleConfig.items}
-          variant="ghost"
-          css={{
-            // @todo drag&drop cursor to adjust numeric value
-            // const cursorUrl = data:image/svg+xml;base64,${btoa(svgCursor)}
-            //cursor: `url(${cursorUrl}), text`,
-            textAlign: "right",
+          contentProps={{ sideOffset: 5 }}
+          value={{
+            name: String(currentValue.value),
+            label: String(currentValue.value),
           }}
```

Comment:
I removed this since it introduces a bug when the selected item is never selected in the dropdown.

Reply:
can you reword? I don't understand the situation

Reply:
let me know once you read it what todo here, lets fix it in a separate PR, because need to merge this now